app/views/submissions/thank_you.html.erb

```diff
@@ -1,17 +1,16 @@
-<div class="thanks">
-  <%= image_tag 'thanks.png' %>
+<div class="l-wrapper-narrow thanks">
+  <%= image_tag 'thanks.svg' %>
   <h1>Thank you for submitting!</h1>
 
-  <p>
+  <p class="small-text">
     <%= link_to 'railsgirls.com/krakow', 'http://railsgirls.com/krakow' %>
   </p>
-  <small>
-    An email saying if you were accepted should be in your inbox after the registration closes.
-    <br/>
-    If you don't receive it, you might have mistyped your address.
-    <br/>
+  <p class="small-text">An email saying if you were accepted should be in your inbox after the registration closes.</p>
+  <p class="small-text">If you don't receive it, you might have mistyped your address.</p>
+  <p class="small-text">
     In that case drop us a line at
     <%= mail_to 'railsgirlskrakow@gmail.com', 'railsgirlskrakow@gmail.com' %>
     to make sure that we can contact you :)
-  </small>
+  </p>
+  </p>
```

Comment:
This closing tag looks like it was left there by mistake (there are two next to each other on the same level)